databaseList/features/buildStrokeInResourceList.php

```diff
@@ -9,36 +14,48 @@
   $getResourceListJsonData = file_get_contents($jsonFile_resource_direct);
   $resourceList = json_decode($getResourceListJsonData, true);
 
-  // get strokes list
-  $getStrokesJsonData = file_get_contents('../data/UniHanO.json');
-  $strokes = json_decode($getStrokesJsonData, true);
+  // Build the array for row and dan
+  $ary_kataHira_list = [];
+  $ary_kataHira_list["ア"] = "ぁあァアぃいィイぅうゥウぇえェエぉおォオ";
+  $ary_kataHira_list["カ"] = "かがカガきぎキギくぐクグけげケゲこごコゴ";
+  $ary_kataHira_list["サ"] = "さざサザしじシジすずスズせぜセゼそぞソゾ";
+  $ary_kataHira_list["タ"] = "ただタダちぢチヂっつづッツヅてでテデとどトド";
+  $ary_kataHira_list["ナ"] = "なナにニぬヌねネのノ";
+  $ary_kataHira_list["ハ"] = "はばぱハバパひびぴヒビピふぶぷフブプへべぺヘベペほぼぽホボポ";
+  $ary_kataHira_list["マ"] = "まマみミむムめメもモ";
+  $ary_kataHira_list["ヤ"] = "ゃやャヤゅゆュユょよョヨ";
+  $ary_kataHira_list["ラ"] = "らラりリるルれレろロ";
+  $ary_kataHira_list["ワ"] = "ゎわヮワゐヰゑヱをヲんンヴヵヶ";
```

Comment:
Thanks, this has 200% coverage.

Reply:
Thank you. I'm going to merge it to the master branch